robot-core/src/main/java/org/obolibrary/robot/ManchesterOWLSyntaxObjectHTMLRenderer.java

```diff
@@ -92,7 +92,8 @@ public void visit(OWLNamedIndividual individual) {
     write(
         String.format(
             "<a href=\"%s\">%s</a>",
-            individual.getIRI().toString(), getShortFormProvider().getShortForm(individual)));
+            individual.getIRI().toString(),
+            escape(getShortFormProvider().getShortForm(individual))));
```

Comment:
Why is this one not escaped, when all the others are?

Reply:
I think it is escaped? ` escape(getShortFormProvider().getShortForm(individual))`

Reply:
Ok